src/gstools/variogram/variogram.py

```diff
@@ -250,18 +263,18 @@ def vario_estimate(
     """
     if bin_edges is not None:
         bin_edges = np.array(bin_edges, ndmin=1, dtype=np.double, copy=False)
-        bin_centres = (bin_edges[:-1] + bin_edges[1:]) / 2.0
+        bin_center = (bin_edges[:-1] + bin_edges[1:]) / 2.0
     # allow multiple fields at same positions (ndmin=2: first axis -> field ID)
     # need to convert to ma.array, since list of ma.array is not recognised
     field = np.ma.array(field, ndmin=2, dtype=np.double, copy=True)
     masked = np.ma.is_masked(field) or np.any(mask)
     # catch special case if everything is masked
     if masked and np.all(mask):
-        bin_centres = np.empty(0) if bin_edges is None else bin_centres
-        estimates = np.zeros_like(bin_centres)
+        bin_center = np.empty(0) if bin_edges is None else bin_center
+        estimates = np.zeros_like(bin_center)
         if return_counts:
-            return bin_centres, estimates, np.zeros_like(estimates, dtype=int)
-        return bin_centres, estimates
+            return bin_center, estimates, np.zeros_like(estimates, dtype=int)
+        return bin_center, estimates
```

Comment:
I find it quite confusing that it's the plural of `bin_edges`, but now the singular of `bin_center`.

Reply:
renamed